tla-types/src/main/scala/at/forsyte/apalache/tla/types/package.scala

```diff
@@ -1,22 +1,134 @@
 package at.forsyte.apalache.tla
 
+/**
+ * Defines a premade instancce of [[at.forsyte.apalache.tla.typecomp.ScopedBuilder ScopedBuilder]], named [[tla]].
+ *
+ * =Importing and using [[tla]]=
+ *
+ * You can access it by importing via
+ * {{{
+ * import at.forsyte.apalache.tla.types._
+ * }}}
+ * It is recommended to import the entire package, instead of just
+ * {{{
+ * import at.forsyte.apalache.tla.types.tla
+ * }}}
+ * due to the reasons outlined below:
```

Comment:
Wasn't quite sure what the reasons are before reading on.

```suggestion
 * It is recommended to import the entire package, instead of just
 * {{{
 * import at.forsyte.apalache.tla.types.tla
 * }}}
 * to enable the implicit conversions described below:
```